wiki/content/deploy/index.md

```diff
@@ -1641,22 +1641,54 @@ The following configuration options are available for Alpha:
 * `--tls_client_auth string` - TLS client authentication used to validate client connection. See [Client authentication](#client-authentication) for details.
 
 ```sh
+# First, create rootca and node certificates
```

Comment:
... and keys.

Reply:
This is resolved in a later commit, I am not sure why this is still unresolved.